src/__test__/actions/assetLogs.actions.test.js

```diff
@@ -0,0 +1,76 @@
+import expect from 'expect';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+
+import constants from '../../_constants';
+
+import loadAssetLogs from '../../_actions/assetLogs.actions';
+
+import assetLogs from '../../_mock/assetLogs';
+
+const {
+  LOAD_ASSET_LOGS_SUCCESS,
+  LOAD_ASSET_LOGS_FAILURE,
+  LOADING_ASSETS_LOGS
+} = constants;
+
+const middleware = [thunk];
+const mockStore = configureMockStore(middleware);
```

Comment:
Why aren't you importing and using the mockStore used in the reducer tests?

Reply:
Using redux thunk middleware based on  [this](https://redux.js.org/recipes/writing-tests#async-action-creators) documentation.